python/common/org/python/types/Float.java

```diff
@@ -169,7 +171,15 @@ public Float(double value) {
         __doc__ = ""
     )
     public org.python.Object __ne__(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("float.__ne__() has not been implemented.");
+        if (other instanceof org.python.types.Int || other instanceof org.python.types.Float ||
```

Comment:
Really? Is that *with* the quotes? float(1.0) should be returning a float, which definitely isn't equal to the *string* 1.0...